### PR TITLE
chore: add url to translatable strings

### LINKED ISSUE #8321: Translations issues second part
Incomplete sentences
- has just scheduled
- on %(app_name)s.
- of your account
- of your account.
- Late payments can be subject to a
- finance fee.
- This is an automatic email from %(app_name)s for organizers of the
- Your
- has received a new session proposal with the following details
- You can view all speaker details on the speaker profile page and the session proposal on the
- To change the status of sessions please go to the
- and click on the status badge of the session.
- Or paste this token in your

No full stop
- You can unsubscribe from emails from this group by unfollowing the group
- You have been invited as a speaker for the session submission "%(session_title)s" for the event

Wrong sentence - what is the real meaning?
- Click on the following link to manage your event

Change "eventyay" with "app name"
- The invoice for your event %(event_name)s on eventyay.com for %(date)s is now available.

### PR DESCRIPTION
**Pull request by progmatic-99**
Fixes #8321 #8327

#### Short description of what this resolves:
Added urls to translatable strings

### CODE REVIEW

**Comment by progmatic-99** [comment]
@mariobehling I have not translated ```session state mails & subjects of all mails``` in this PR. As that will need changes in more than 1 python file and needs to be discussed first.

**Comment by progmatic-99** [comment]
I have also solved [this](https://github.com/fossasia/open-event-server/issues/8327) in this PR only.

**Comment by mariobehling** [comment]
Ok, thanks a lot. Looking forward to the remaining changes in the next PR.

File: app/templates/email/monthly_payment_pre_due_email.html [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
 <br><br>
-{{ _('Pay Now at') }}: <a href='{{ payment_url }}'>{{ payment_url }}</a>
+{{ _('Pay Now at: %(payment_url)s', payment_url = '<a href="%s">%s</a>' % (payment_url, payment_url)) | safe }}
 <br><br>
-{{ _('Late payments can be subject to a') }} 5% {{ _('finance fee.') }}
+{{ _('Late payments can be subject to a %(percentage)s finance fee.', percentage='5%') }}
 <br><br>
-{{ _('A detailed invoice is available in') }} <a href='https://eventyay.com/account/billing/invoices/'>{{ _('the billing area') }}</a> {{ _('of your account.') }}
-{{ _('If you have any questions about invoices, please find more information on our FAQ at') }} <a href='https://support.eventyay.com'>https://support.eventyay.com</a>.
+{{ _('A detailed invoice is available in %(open_link)sthe billing area%(close_link)s of your account.', open_link = '<a href="https://eventyay.com/account/billing/invoices/">', close_link = '</a>') | safe }}
```
[the remark this answers is not in the document]
Comment (pc-beast) [reply app/templates/email/monthly_payment_pre_due_email.html:13]: It was here previously, but we may want to change ```eventyay.com``` to ```frontend_url``` for the people using their own instance of our app or for development and testing purposes. This hardcoded value works for our instance, not for other instances say ```test.eventyay.com```

**Pull request merged**